src/Version.php

```diff
@@ -23,7 +23,7 @@ final class Version
      *   Stable — 3.0.0
      *   Development — 3.1.0 alpha 1
      */
-    public const VERSION = '5.2.0 RC 1';
+    public const VERSION = '5.2.1';
```

Comment:
The version must be changed in a separate PR after the latest release. Right?

Reply:
Actually, this was missed for the 5.2.0 release and I am directly releasing 5.2.1 when this is merged. The test fixes are just a bonus to get it working.